frontend/src/components/jobs/result/utils/sections.jsx

```diff
@@ -90,8 +103,13 @@ export function JobActionsBar({ job, refetch }) {
     playbooks: job.playbooks_to_execute.map((x) => x.value),
   };
 
+  const handleRetry = async () => {
+    const jobId = await createJob(formValues).then(refetch);
+    setTimeout(() => navigate(`/jobs/${jobId[0]}`), 500);
```

Comment:
the status code error for timeout should be 504, not 500

Reply:
@mlodic isn't 500 the time in ms after which the callback func is executed?